WordPress/Classes/Models/ManagedAccountSettings+CoreDataProperties.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+import CoreData
+
+extension ManagedAccountSettings {
```

Comment:
I still find a bit weird that the properties that define an object are in an extension, and the helper methods in the class